Readme.md

```diff
@@ -81,6 +82,7 @@ modules:
         - thedevelopers
       create: true
       delete: true
+      reset: true
```

Comment:
The doc needs a bit more explanation here.
First, I think the default value should be `false`, because `true` would mean all users get a new password every time userplex is ran.
Second, there should be a paragraph that clearly states what `reset` is, based on the above. It's not as obvious as create and delete. Does it remove groups and access keys? etc...